gui/mozregui/global_prefs.py

```diff
@@ -24,6 +25,8 @@ def save_prefs(options):
     settings.update({
         'persist': options['persist'] or '',
         'http-timeout': options['http_timeout'],
+        'persist-size-limit': int(abs(float(options.persist_size_limit))
```

Comment:
hum, thinking more about it, I think we should do that conversion from GiB to bytes in bisection.py, along with:

```
persist_limit = global_prefs['persist_size_limit']
```